subiquitycore/ui/utils.py

```diff
@@ -236,7 +236,7 @@ def screen(rows, buttons, focus_buttons=True, excerpt=None, narrow_rows=False):
     if isinstance(rows, list):
         rows = ListBox(rows)
     if narrow_rows:
-        rows = Padding.center_63(rows)
+        rows = Padding.center_76(rows)
```

Comment:
@mwhudson I think you mean this, no? center_63 is what most centered view forms try to use, whilst center_76 is actually the width of the progress-bar below.

I wonder if we can actually re-evaluate the screen estate and make filesystem screens not use so much width. E.g. popups can open to the left, on top of the screen, instead of to the right.